src/backend/dx11/src/factory.rs

```diff
@@ -453,7 +459,7 @@ impl core::Factory<R> for Factory {
             &core::ShaderSet::Tessellated(ref vs, ref hs, ref ds, ref ps) => {
                 let (vs, hs, ds, ps) = (vs.reference(fh), hs.reference(fh), ds.reference(fh), ps.reference(fh));
               
-			  populate_info(&mut info, Stage::Vertex,   vs.reflection);
+              populate_info(&mut info, Stage::Vertex,   vs.reflection);
```

Comment:
Yes, it was originally intended with tabs. The default for the project is AFAIK "4 spaces"